src/Albany_Utils.cpp

```
@@ -564,3 +564,19 @@ void Albany::assert_fail(std::string const& msg) {
   std::cerr << msg;
   abort();
 }
+
+namespace Albany {
+
+BuildType build_type(const BuildType value)
+{
+  static bool initialized_ = false;
+  static BuildType value_ = value;
+  if (!initialized_ && (value != BuildType::None)) {
```

Should we throw an exception if no build type is selected?